services/integration/src/models/constitution/PreviousInformation.model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const previousInformationSchema = new Schema({
+  projectId: {
+    type: String,
+    required: [true, 'Project Id obligatory field'],
```

Comment:
``projectId` is an required field` should be right message here, please use this patter for every required field

Reply:
change is made